spec/api/v1/users_spec.rb

```diff
@@ -0,0 +1,68 @@
+require 'rails_helper'
+
+RSpec.describe V1::Users do
+  let(:api_token) { create :api_token }
+
+  describe '#GET /api/v1/users' do
+
+    context 'when api token is invalid' do
+      let(:api_token) { double :api_token, token: 'foo' }
+
+      before do
```

Comment:
I saw that you are adding a `before` block in all your tests, but its not necessary (maybe you are following this as rule). They are useful when you need to share some setup between tests. As you are using it only for a single test, it's fine to add the code direct in the `it`:

```ruby
it 'returns a authorization error' do
  get '/api/v1/users', api_key: api_token.token

  expect(response.body).to match(/Invalid token/)
end
```